<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainApp</class>
 <widget class="QMainWindow" name="MainApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DBSP Project Commander</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(13, 76, 128);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>950</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 11pt &quot;Segoe UI&quot;;</string>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>FIO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DEPARTMENT</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>PHONE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>START_D</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="photo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>331</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="welcome">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>331</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Личный кабинет</string>
    </property>
   </widget>
   <widget class="QLabel" name="fio">
    <property name="geometry">
     <rect>
      <x>361</x>
      <y>30</y>
      <width>601</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>fio</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>370</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Test_info</string>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>621</x>
      <y>370</y>
      <width>249</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>time</string>
    </property>
   </widget>
   <widget class="QLabel" name="date">
    <property name="geometry">
     <rect>
      <x>361</x>
      <y>370</y>
      <width>249</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_to_home">
    <property name="geometry">
     <rect>
      <x>621</x>
      <y>420</y>
      <width>249</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>time_to_home</string>
    </property>
   </widget>
   <widget class="QLabel" name="date_2">
    <property name="geometry">
     <rect>
      <x>361</x>
      <y>420</y>
      <width>249</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Отработано сегодня:</string>
    </property>
   </widget>
   <widget class="QLabel" name="tabelnomer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>249</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>tabelNomer</string>
    </property>
   </widget>
   <widget class="QLabel" name="mobileInfo">
    <property name="geometry">
     <rect>
      <x>361</x>
      <y>315</y>
      <width>219</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Мобильный телефон</string>
    </property>
   </widget>
   <widget class="QLabel" name="departmentInfo">
    <property name="geometry">
     <rect>
      <x>361</x>
      <y>94</y>
      <width>63</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Отдел</string>
    </property>
   </widget>
   <widget class="QLabel" name="carInfo">
    <property name="geometry">
     <rect>
      <x>361</x>
      <y>204</y>
      <width>112</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Транспорт</string>
    </property>
   </widget>
   <widget class="QLabel" name="dolzhnostInfo">
    <property name="geometry">
     <rect>
      <x>361</x>
      <y>149</y>
      <width>116</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Должность</string>
    </property>
   </widget>
   <widget class="QLabel" name="emailImfo">
    <property name="geometry">
     <rect>
      <x>361</x>
      <y>260</y>
      <width>70</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Имейл</string>
    </property>
   </widget>
   <widget class="QLabel" name="dolzhnost">
    <property name="geometry">
     <rect>
      <x>621</x>
      <y>149</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>label_11</string>
    </property>
   </widget>
   <widget class="QLabel" name="car">
    <property name="geometry">
     <rect>
      <x>621</x>
      <y>204</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>label_15</string>
    </property>
   </widget>
   <widget class="QLabel" name="email">
    <property name="geometry">
     <rect>
      <x>621</x>
      <y>260</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>label_9</string>
    </property>
   </widget>
   <widget class="QLabel" name="department">
    <property name="geometry">
     <rect>
      <x>621</x>
      <y>94</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>label_12</string>
    </property>
   </widget>
   <widget class="QLabel" name="mobile">
    <property name="geometry">
     <rect>
      <x>621</x>
      <y>315</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>label_7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendMessage">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>280</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>80</y>
      <width>461</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connect">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>30</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outMessage">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>280</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>490</y>
      <width>91</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 76, 128);</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/img/2020.jpg</normaloff>:/img/2020.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>141</width>
      <height>150</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>490</y>
      <width>91</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 190, 152);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/img/2024.webp</normaloff>:/img/2024.webp</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>490</y>
      <width>91</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(187, 39, 73);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/img/2023.jpg</normaloff>:/img/2023.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Попробуйте новый стиль</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>370</y>
      <width>261</width>
      <height>281</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>242</width>
       <height>879</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="depAll">
        <property name="text">
         <string>Все отделы</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="d01">
        <property name="text">
         <string>Отдел 01</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="d09">
        <property name="text">
         <string>Отдел 09</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="d13">
        <property name="text">
         <string>Отдел 13</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="d14">
        <property name="text">
         <string>Отдел 14</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="d20">
        <property name="text">
         <string>Отдел 20</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="d21">
        <property name="text">
         <string>Отдел 21</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="d22">
        <property name="text">
         <string>Отдел 22</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="d23">
        <property name="text">
         <string>Отдел 23</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="d24">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Отдел 24</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="d25">
        <property name="text">
         <string>Отдел 25</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="d29">
        <property name="text">
         <string>Отдел 29</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="d31">
        <property name="text">
         <string>Отдел 31</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="d34">
        <property name="text">
         <string>Отдел 34</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="d63">
        <property name="text">
         <string>Отдел 63</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="d65">
        <property name="text">
         <string>Отдел 65</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="d73">
        <property name="text">
         <string>Отдел 73</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="d75">
        <property name="text">
         <string>Отдел 75</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="d81">
        <property name="text">
         <string>Отдел 81</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="d82">
        <property name="text">
         <string>Отдел 82</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="d89">
        <property name="text">
         <string>Отдел 89</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="d91">
        <property name="text">
         <string>Отдел 91</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="d92">
        <property name="text">
         <string>Отдел 92</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="d95">
        <property name="text">
         <string>Отдел 95</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>1000</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Отдел:</string>
    </property>
   </widget>
   <zorder>date</zorder>
   <zorder>welcome</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>time</zorder>
   <zorder>dolzhnost</zorder>
   <zorder>mobileInfo</zorder>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>emailImfo</zorder>
   <zorder>mobile</zorder>
   <zorder>date_2</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>tabelnomer</zorder>
   <zorder>car</zorder>
   <zorder>dolzhnostInfo</zorder>
   <zorder>photo</zorder>
   <zorder>departmentInfo</zorder>
   <zorder>time_to_home</zorder>
   <zorder>carInfo</zorder>
   <zorder>email</zorder>
   <zorder>pushButton</zorder>
   <zorder>department</zorder>
   <zorder>fio</zorder>
   <zorder>tableWidget</zorder>
   <zorder>sendMessage</zorder>
   <zorder>textBrowser</zorder>
   <zorder>connect</zorder>
   <zorder>outMessage</zorder>
   <zorder>scrollArea</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>lineEdit</zorder>
   <zorder>label_3</zorder>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionProducts"/>
   <addaction name="actionMessages"/>
   <addaction name="actionPersonnel"/>
   <addaction name="actionAccount"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionProducts">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/icons8-папка-расширений-50.png</normaloff>:/img/icons8-папка-расширений-50.png</iconset>
   </property>
   <property name="text">
    <string>Products</string>
   </property>
   <property name="toolTip">
    <string>Products</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionMessages">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/icons8-конверт-50.png</normaloff>:/img/icons8-конверт-50.png</iconset>
   </property>
   <property name="text">
    <string>Messages</string>
   </property>
   <property name="toolTip">
    <string>Messages</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionPersonnel">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/icons8-пользователи-50.png</normaloff>:/img/icons8-пользователи-50.png</iconset>
   </property>
   <property name="text">
    <string>Personnel</string>
   </property>
   <property name="toolTip">
    <string>Personnel</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionAccount">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/personal_account.png</normaloff>:/img/personal_account.png</iconset>
   </property>
   <property name="text">
    <string>Account</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
